bot/cogs/help_channels.py

```diff
@@ -210,6 +210,9 @@ def create_name_queue(self) -> deque:
         log.trace("dormant command invoked; checking if the channel is in-use.")
         if ctx.channel.category == self.in_use_category:
             if await self.dormant_check(ctx):
+                with suppress(KeyError):
+                    del self.help_channel_claimants[ctx.channel]
```

Comment:
I'd always delete when a channel is main dormant - not just in the dormant command. But it's not important.

Reply:
Is there anything that's always ran when moving except the `move_to_dormant` itself that could be used?
Not losing out on much here through the few extra dicts pairs and as I mentioned in a previous conversation I think it makes more sense to just leave the move method to only do stuff directly relevant to it when possible with the command itself grouping the actions now.

Reply:
No, I don't think so. But my point wasn't to move the delete just for the sake of moving it. It was so the dictionary would also be cleared when a move happens automatically. Really, the only benefit is that it prevents it from accumulating useless values. However, I imagine the memory footprint will still be insignificant which is why I said it's not important

Reply:
> However, I imagine the memory footprint will still be insignificant which is why I said it's not important

That's my intuition as well. Since we're keeping a mapping of channels to members, we should have a maximum of "max help channels" mappings in the dictionary under normal circumstances. Even if all channels go dormant with command invocation, the number of additional object references should be manageable.